### DISCUSSION TITLE
怎么在开发模式下用本地文件打包

### DISCUSSION THREAD

**Discussion by JadeVane** [tw93/Pake#657] [Q&A] [answered]
看了一下使用pake-cli的方法，但没人问到yarn直接开发的方法

想打包一个本地文件夹的内容，并在 `pake.json` 中配置了路径

```json
{
  "windows": [
    {
      "url": "./yaaw-zh-hans/index.html",
      ...
      "url_type": "web"
    }
  ]
}
```

但打包完显示找不到文件，试过把文件放在了 src-tauri 下和根目录都没用，应该放在那里？还是说这样的url写法不对？

**Comment by Tlntin** [comment, marked as answer]
参考这个[wiki](https://github.com/tw93/Pake/wiki/Pake%E6%89%93%E5%8C%85%E9%9D%99%E6%80%81%E6%96%87%E4%BB%B6%E7%A4%BA%E4%BE%8B)
然后看看它的文件内容变化。
